lib/modules/filteReddit.js

```diff
@@ -769,21 +760,16 @@ modules['filteReddit'] = {
 			(RESUtils.currentSubreddit('all')) || (RESUtils.currentDomain()) || (RESUtils.currentMultireddit('me/f/all')),
 			onSavedPage = modules['filteReddit'].excludeSaved.test && modules['filteReddit'].excludeSaved.test(location.href);
 
-		Array.prototype.slice.call(entries).forEach(function(entry) {
-			var postSubreddit, currSub;
+		Array.prototype.slice.call(things).forEach(function(thing) {
```

Comment:
You probably don't need the `Array.prototype.slice.call` part, since `.things` should return an array.